libs/blocks/global-navigation/global-navigation.css

```diff
@@ -551,7 +551,8 @@ header.global-navigation {
   }
 
   .feds-menu-container {
-    min-width: 100%;
+    max-width: 1200px;
```

Comment:
There's already a variable for this value, `--feds-minWidth-menu`

Reply:
Added `--feds-minWidth-menu` variable for width.